script.js

```diff
@@ -0,0 +1,37 @@
+/* Julie Rizzino
+script.js
+INFO 2134
+Thoendel
+3/15/20 */
+window.addEventListener('load', function() {
+    /* Write your solution between this comment */
+var selectAge = document.getElementById("age");
+//name variable selectAge to represent the age select option
+var ages;
+//name varaible ages to represent the ages to choose from 0-100
+for (let i = 0; i <= 100; i++) {
+  ages += "<option>" + i + "</option>";
+}
+//for loop to fill the ages from 0-100
+selectAge.innerHTML = ages;

```

Comment:
this is why you didn't keep the original option. What if instead of building a string - you built objects in the DOM and then appended them. That is the approach the book was showing to solve these kinds of issues. Something like:
        //create child element object
        let child = this.document.createElement("option");
        //set value
        child.value = i;
        //set text
        child.text = i.toString();
        //append child to the select
        a.appendChild(child);

Where a is the age object - already containing the original option.